mozbench/mozbench.py

```diff
@@ -36,6 +36,105 @@
 headers = None
 results = None
 
+class ResultRecorder(object):
+
+    def __init__(self):
+        self._platform = 'unknown'
+        self._osVersion = 'unknown'
+        self._processor = 'unknown'
+        self._currentBrowser = None
```

Comment:
Class members should have underscores in their names, so this should be current_browser.